app/contracts/project_life_cycle_steps/base_contract.rb

```diff
@@ -31,9 +31,7 @@ class BaseContract < ::ModelContract
     validate :select_custom_fields_permission
     validate :consecutive_steps_have_increasing_dates
 
-    def valid?(context = :saving_life_cycle_steps)
-      super
-    end
+    def valid?(context = :saving_life_cycle_steps) = super
```

Comment:
@dombesz genial! - perhaps we can add a Rubocop to prefer endless functions?

Reply:
Thanks @myabc ! Added it here https://github.com/opf/openproject/pull/17707 .